developer-docs-site/docs/cli-tools/aptos-cli-tool/install-aptos-cli.md

```diff
@@ -10,12 +10,13 @@ This document describes how to install the `aptos` CLI tool. See [Use Aptos CLI]
 
 ## Install precompiled binary (easy mode)
 
-* Navigate to the [release page](https://github.com/aptos-labs/aptos-core/releases) for Aptos CLI.
-* Download the latest release for your platform.
-* Place this at a location for you to run it e.g. `~/bin/aptos` in Linux.
-* On Linux and Mac, make this executable `chmod +x ~/bin/aptos`.
-* Now type `~/bin/aptos help` to read help instructions.
-* If you want you can add `~/bin` to your path in your appropriate `.bashrc` or `.zshrc` for future use.
+1. Navigate to the [release page](https://github.com/aptos-labs/aptos-core/releases?q=cli&expanded=true) for Aptos CLI.
+2. From the latest release section, download the binary zip file for your platform. The binary zip files contain the platform name in the filename, i.e., `aptos-cli-0.2.3-MacOSX-x86_64.zip` for the MacOS binary and `aptos-cli-0.2.3-Ubuntu-x86_64.zip` for the Linux binary.
```

Comment:
considering these versions change so frequently, do we want to be explicit with which version?

Reply:
This is just to show them an example of the filename w/ platform name in it, not pointing to a specific version. But the readers might come to the same conclusion as you did, so I will tweak it.